superbench/benchmarks/micro_benchmarks/gpu_sm_copy_bw_performance.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the GPU SM Copy Bandwidth Performance benchmark."""
+
+import os
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, ReturnCode
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class GpuSmCopyBwBenchmark(MicroBenchmarkWithInvoke):
+    """The GPU SM copy bandwidth performance benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'gpu_sm_copy'
+
+        self.__result_tags = []
+        self.__num_numa_nodes_in_system = len(
+            [x for x in next(os.walk('/sys/devices/system/node/'))[1] if x.startswith('node')]
+        )
+        self.__default_numa_nodes = list(range(self.__num_numa_nodes_in_system))
+        self.__num_gpus_in_system = 0
+        try:
+            self.__num_gpus_in_system = len([x for x in next(os.walk('/sys/class/drm/'))[1] if x.startswith('card')])
```

Comment:
@guoshzhao  how can we handle it?

Reply:
I think we can use os.getenv('NUMA_NODES') and os.getenv('CUDA_VISIBLE_DEVICES') to get the list?

Reply:
Now let executor to force `numactl` and `CUDA_VISIBLE_DEVICES` from outside.